Makefile

```diff
@@ -0,0 +1,143 @@
+.DEFAULT_GOAL := help
```

Comment:
Might be worth resyncing against craft-providers Makefile to address all the feedback I got on this one :D